src/lib/features/traffic-data-usage/traffic-data-usage-store.test.ts

```diff
@@ -20,6 +21,10 @@ afterAll(async () => {
     await db.destroy();
 });
 
+beforeEach(async () => {
+    await trafficDataUsageStore.deleteAll();
+});
```

Comment:
We were calling this in all the tests, so it's probably better to extract it so that we don't need to remember.